modules/web3/hooks/useTotalDai.tsx

```diff
@@ -0,0 +1,29 @@
+import useSWR from 'swr';
+import { useContracts } from 'modules/web3/hooks/useContracts';
+
+export const useTotalDai = (): any => {
+  const { vat } = useContracts();
+
+  const { data, error } = useSWR('total-dai', async () => {
+    return await vat.debt();
+  });
+
+  return {
+    data,
+    loading: !error && !data,
+    error
+  };
+};
+
+// temp comment: dai.js implementation for reference
+
+// async getTotalDai() {
```

Comment:
I believe this is the total Dai in the DSR, `vat.debt()` is the total dai in circulation (I compared the result with system stats on prod and in daistats, and it's the same).